README.md

```diff
@@ -1,2 +1,29 @@
 # Hello_Microverse
 First Project at Microverse Program - Day 2
+
+#Description the project.
+
+Set up a 'Hello Microverse' project it is a Project of the second Day at Microverse Program
+
+#Built With
+
+HTML, CSS
+Github flow
+VSCode
+#Author
+
+👤 Javier Salinas
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+#Show your support
+
+Give a ⭐️ if you like this project!
+
+📝 License
+
+This project is MIT licensed.
+
```

Comment:
- [ ] Please use the updated Readme Template as recommended in the project requirements, you can use the template to document your projects , [you can get it from here ](https://github.com/microverseinc/readme-template) copy the content of Readme.md file then customise it to your project :+1: